frontend/client/views/settings/index.vue

```diff
@@ -483,6 +563,22 @@
         this.close()
       },
 
+      resetUserTriggerToken () {
+        this.$http
+          .get('/api/v1/user/' + this.selectUser.username + '/reset-trigger-token')
```

Comment:
In my opinion, it should be PUT/POST/PATCH since this is an update operation?

Reply:
Agreed. Get is not very nice. :)